src/common/selectors/index.js

```diff
@@ -1,8 +1,14 @@
 import { createSelector } from 'reselect';
+import pick from 'lodash/pick';
 
 import { parseGitHubRepoUrl } from '../helpers/github-url';
 
+const getRepositoriesIndex = state => state.repositories.ids;
+const getRepositories = state => state.entities.repos;
+const getRepositoryOwners = state => state.entities.owners;
 const getSnaps = state => state.snaps;
+const _getSnapsIndex = state => state.snaps.ids;
+const _getSnaps = state => state.entities.snaps;
```

Comment:
underscored because they're the newer pattern and would need to refactor out the non underscore version in the work to tackle snaps.